templates/compositions/ec-component-content-item/content-item.html.twig

```diff
@@ -0,0 +1,94 @@
+{% spaceless %}
+
+{#
+  /**
+   * @file
+   * Theme implementation for a ECL Content Item.
+   */
+  Parameters:
+    - "content_item" (associative array) (default: predefined structure): format:
+        {
+          description: '', (string) (default: ''): Description of Content Item
+          meta: [], (array) (default: []): Meta's for the Content Item
+          title: {}, (associative array) (default: {}): Predefined structure compatible with Link component. If Content Item type is a 'tile', only label property is required.
+          image: '', (string) (default: ''): Url/path to background image (non required if Content Item type is a 'tile')
```

Comment:
Is this `tile` or `title`? Either way I don't see any reference of it in the implementation.

Reply:
Legacy that I need remove.